src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -241,6 +277,12 @@ try {
 
 ###### `routeData`
 
+<p>
+
+**Type:** `RouteData`<br />
+**Default:** `app.match(request)`
```

Comment:
It seems the default is the return type of `app.match(request)` and not `app.match(request)`. But I couldn't find the value... Is there a way to search on a branch other than the main one?

Reply:
Should this now be `IntegrationRouteData`?

Reply:
I hadn't paid attention to the sentence below, so I understand your question... I checked again:
* By switching back to the `next` branch: the type is still `RouteData`.
* So I checked `IntegrationRouteData` to make sure it's not an alias: this is a [smaller version of `RouteData`](https://github.com/withastro/astro/blob/6860beb995cab405e6c3775b270bbe9e86fb561a/packages/astro/src/types/public/integrations.ts#L247).

So I don't think so, `RouteData` type still exists and it seems to be the one expected. And I guess the sentence below remains valid: we provide a full `RouteData` object which is passed to `integrationRouteData` in its smaller form.
But a check from someone more informed on how it works may be useful, this is marked as `**Advanced API**: you probably do not need to use this.`

Reply:
ok, then sounds good! Thank you!